src/Core/Commander/Providers/Fetcher.js

```diff
@@ -2,6 +2,8 @@ import { TextureLoader } from 'three';
 
 const textureLoader = new TextureLoader();
 
+textureLoader.setCrossOrigin('');
```

Comment:
Yes, but the second one is explicit (ie: doesn't require the reader to know that `''` maps to `'anonymous'`)